packages/@tinacms/toolkit/src/packages/fields/plugins/MdxFieldPlugin/updated/plate/plugins/ui/components.tsx

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 
 import React from 'react'
 import { ELEMENT_HR } from '@udecode/plate-horizontal-rule'
+import { ELEMENT_BR } from '@udecode/plate-break'
```

Comment:
I was getting a type error here and could not find any docs about using this package like this. I had better luck using the `"br"` string directly (although this might not be totally correct either )